src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
+$size: 100px;
+$content-day: '';
+$start-days: (
+  'mon': 0,
+  'tue': $size + 1px,
+  'wed': $size * 2 + 2px,
+  'thu': $size * 3 + 3px,
+  'fri': $size * 4 + 4px,
+  'sat': $size * 5 + 5px,
+  'sun': $size * 6 + 6px,
+);
+$month-length: (
+  '28': 28,
+  '29': 29,
+  '30': 30,
+  '31': 31,
+);
+
+@mixin flex($flex, $justify-content, $align-items) {
+  display: $flex;
+  justify-content: $justify-content;
+  align-items: $align-items;
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: $content-day + #{$i};
+  }
+}
+
+@each $day, $margin in $start-days {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: #{$margin};
+    }
+  }
+}
+
+@each $length, $num in $month-length {
+  .calendar--month-length-#{$length} {
+    @for $i from 1 through $num {
+      .calendar__day {
+        visibility: hidden;
+      }
+
+      .calendar__day:nth-child(#{$i}) {
+        visibility: visible;
+      }
+    }
+  }
+}
+
 body {
+  @include flex(flex, center, center);
+
+  flex-direction: column;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  @include flex(flex, left, center);
```

Comment:
The `flex` parameter in the `flex` mixin is being used as a display property value. Ensure that the correct value is passed to the mixin, as it might not be the intended use.